build/webpack.common.js

```diff
@@ -39,7 +38,7 @@ const withHTMLPlugin = returnPluginWithConfig(HtmlPlugin, {
   template: `${BOOTSTRAP_DIR}/index.html`, // source template
   title: UI_TITLE, // HTML title
   favicon: `${IMAGES_DIR}/favicon.ico`, // favicon for this page
-  inject: 'head', // any/all built content linked into HTML head element
+  inject: true,
```

Comment:
If script loads from `<head>` it's before the DOM exists (so react can't mount elements)

Reply:
TIL - can I catch up with you re this/the windows involved?